plugin/src/main/scala/migrate/CommandStrings.scala

```diff
@@ -27,7 +27,18 @@ object CommandStrings {
   val migrateScalacOptionsDetailed =
     s"""|$migrateScalacOptionsCommand <projectId>
         |
-        | Print the migrated scalacOptions for Scala 3 so you can update you scalacOptions
+        |Print the migrated scalacOptions for Scala 3 so you can update you scalacOptions
+        | 
+        |
+        |""".stripMargin
+
+  val migrateLibs      = "migrate-libs"
+  val migrateLibsBrief = (migrateLibs, "Find and show the new versions of libs in order to migrate to Scala 3")
+  val migrateLibsDetailed =
+    s"""|$migrateLibs <projectId>
+        |
+        |Using coursier, we find for each lib the new versions that were publish for scala 3.
+        |If the lib is not yet published for scala 3, we show the available versions for 2.13, in case this lib doesn't contain Macros.
```

Comment:
```suggestion
        |If the lib is not published for Scala 3 yet, versions compatible with 2.13 are reported, in case the lib does not contain macros.
```